main/res/values/strings.xml

```diff
@@ -345,7 +345,8 @@
     <string name="menu_history">History</string>
     <string name="menu_filter">Filter</string>
     <string name="menu_scan_geo">Scan geocode</string>
-    <string name="menu_lists_pocket_queries">Lists / pocket queries</string>
+    <string name="menu_lists_pocket_queries">Pocket queries</string>
+    <string name="menu_lists_bookmarklists">Bookmarklists</string>
```

Comment:
IMHO, there should be a space between "bookmark" and "lists" in English

Reply:
Or maybe call it just `Bookmarks`?

Reply:
I just have browsed a bit through their website and app, and it seems there's not a single place where they use the word "bookmark" anymore.

We still need "bookmark" to distinguish between our offline lists, but at least this means we cannot strip the word "lists" 

![Screenshot_20210912_154235_com groundspeak geocaching intro_copy_432x1028](https://user-images.githubusercontent.com/64581222/132990028-918e8788-0c52-435d-9952-9af435e1709f.jpg)

Reply:
Ok, changed it to bookmark lists